File: IP Requesting 1.0/ip_request_app.py
import tkinter as tk
import requests

class IpRequestApp:
    def __init__(self, master):
        self.master = master
        master.title("IP Tools - Requesting")

        # Create input field
        self.ip_label = tk.Label(master, text="Enter IP Address:")
        self.ip_label.grid(row=0, column=0)
        self.ip_entry = tk.Entry(master)
        self.ip_entry.grid(row=0, column=1)

        # Create Request button
        self.request_button = tk.Button(master, text="Request", command=self.request_ip)
        self.request_button.grid(row=1, column=0, columnspan=2)

        # Create Result label
        self.result_label = tk.Label(master, text="")
        self.result_label.grid(row=2, column=0, columnspan=2)

    def request_ip(self):
        # Get input value
        ip_address = self.ip_entry.get()

        # Send request to ipify API
        try:
            response = requests.get(f"https://api.ipify.org/?format=json&ipAddress={ip_address}")
            data = response.json()

            # Display IP address
            self.result_label.config(text=f"Your IP address is: {data['ip']}")
        except requests.exceptions.RequestException as e:
            self.result_label.config(text=f"Error: {e}")

# Create GUI instance
root = tk.Tk()
app = IpRequestApp(root)

# Run GUI
root.mainloop()

File: IP Requesting 1.0/ip_request_app.java
import javax.swing.*;
import java.awt.*;
import java.awt.event.ActionEvent;
import java.awt.event.ActionListener;
import java.io.IOException;
import org.jsoup.Jsoup;
import org.jsoup.nodes.Document;
import org.jsoup.nodes.Element;

public class IpRequestApp extends JFrame {

    private JTextField ipField;
    private JLabel resultLabel;

    public IpRequestApp() {
        setTitle("IP Tools - Requesting");
        setSize(400, 150);
        setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
        setLayout(new BorderLayout());

        JPanel topPanel = new JPanel();
        JLabel ipLabel = new JLabel("Enter IP Address:");
        ipField = new JTextField(20);
        JButton requestButton = new JButton("Request");
        topPanel.add(ipLabel);
        topPanel.add(ipField);
        topPanel.add(requestButton);

        resultLabel = new JLabel("");
        resultLabel.setHorizontalAlignment(JLabel.CENTER);

        requestButton.addActionListener(new ActionListener() {
            public void actionPerformed(ActionEvent e) {
                requestIp();
            }
        });

        add(topPanel, BorderLayout.NORTH);
        add(resultLabel, BorderLayout.CENTER);
    }

    private void requestIp() {
        String ipAddress = ipField.getText();
        String resultText = "";

        try {
            Document document = Jsoup.connect("https://api.ipify.org/?format=json&ipAddress=" + ipAddress).get();
            Element element = document.selectFirst("pre");

            if (element != null) {
                String jsonText = element.text();
                resultText = "Your IP address is: " + jsonText;
            } else {
                resultText = "Error: Invalid response from API";
            }
        } catch (IOException e) {
            resultText = "Error: " + e.getMessage();
        }

        resultLabel.setText(resultText);
    }

    public static void main(String[] args) {
        SwingUtilities.invokeLater(new Runnable() {
            public void run() {
                IpRequestApp app = new IpRequestApp();
                app.setVisible(true);
            }
        });
    }
}
